index.html

```diff
@@ -132,95 +132,56 @@ <h2>Data Dashboard Healthcare</h2>
                     </div>
 
                     <div class="project">
-                        <h2>Data Dashboard Healthcare</h2>
-                        <p>A daily selection of privately personalized reads
-                            no accounts or sign-ups required. has been the industry's standard
-                        </p>
+                        <h2></h2>
+                        <p></p>
                         <div class="languages">
 
                             <ul>
-                                <li class="project-language"> html
-                                <li>
-                                <li class="project-language">bootstrap
-                                <li>
-                                <li class="project-language"> Ruby
-                                <li>
                             </ul>
                         </div>
                         <button class="project-btn" onclick="modal(1)">See project</button>
 
                     </div>
                     <div class="project">
-                        <h2>Profesional Art Printing Data</h2>
-                        <p>A daily selection of privately personalized reads
-                            no accounts or sign-ups required. has been the industry's standard
+                        <h2></h2>
+                        <p>
                         </p>
                         <div class="languages">
```

Comment:
- [ ] As requested by the first reviewer all the HTML for the project's list including the div for each project, the see project button, and the languages list ... all the elements should be generated dynamically using javascript. This will make adding a new project to your Portfolio in the future easier. Instead of updating both the HTML and the project's array, you'll only have to add the information in the project's array. Kindly fix the issue.

Reply:
I apologize I think the pull request being messed up led you to open an old commit instead ! the only commit here on this pull request by the link : https://github.com/basnetlaxmi/my-portfolio/blob/9902eef5ff7fddddd719cf6b6f64b296a3840535/index.html

**for example** :

![wrong commit file](https://user-images.githubusercontent.com/83329154/132078423-00853269-59c0-4f3f-94ce-1edebd65f685.png)


The latest commit on this pull request has all the issues fixed except for only 2 boxes ommited on purpose (one is animated and we ignored that, and the other is part of the project list but we ommited it because it doesn't have a figma model ". I only mentioned the previous pull request because I wanted to show the previous code review changes and the flow of the task

Reply:
- I'm pretty sure I checked the changes in this PR. The code that I commented, What's red is what you removed, and what's green is the new changes you made in this branch.